### PR TITLE
Initial frontend setup

### PR DESCRIPTION
**Pull request by Bailey-Pearson**
### <a href="https://trello.com/c/oI5Ku2Dd/23-shared-components-2-points">Shared Components (2 Points)</a>

- Added React Router for each page
- Simplified App.js to be usable for any pages out of the box
- Made a `/styles` page featuring our colors/fonts/libraries and shared components
- Installed reactstrap and material icons
- Updated App.css to contain some existing styles

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,88 @@
...
+.banner-text{
+    font-family: 'Montserrat', sans-serif;
+    font-size: 30px;
+    color: #FFFFFF; 
+    margin: 0 !important;
+}
+
+.game-window{
+    width: 70vw; 
+    height: 70vh;
```
Comment (jeremydelacruz) [inline src/App.css:73]: Just curious, what do these units do for us? I'm assuming it's like a 70%?

File: src/pages/Styles.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        return (
+            <div className="App">
+                <div className="Page">
+                    <div style={{height: 'fitContent', width: '70%', borderStyle: 'solid', padding: '20px', borderRadius: '20px', backgroundColor: '#FFFF'}}>
+                        <div>
+                            <div style={{textAlign: 'center'}} >
+                                <h1> Style References </h1>
+                            </div>
+                            <i>
+                                This is a reference page for the styles we will be using for the cite. Reference this for questions on how to set up
```
Comment (jeremydelacruz) [inline src/pages/Styles.js]:
```suggestion
                                This is a reference page for the styles we will be using for the site. Reference this for questions on how to set up
```

**Review by jeremydelacruz — APPROVED** [review]
This all looks really good to me. The components look really straightforward and easy to use

File: src/App.css (hunk 1 above)
Comment (jakeod99) [reply src/App.css:73]: `vh` and `vw` are units that correspond to the percentage based on the size of the viewport

Comment (Bailey-Pearson) [reply src/App.css:73]: Yeah the react components can be real weird with percentage based height so sometimes it’s just easier to use vh and vw

File: src/tests/App.test.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import App from '../App';
```
Comment (jakeod99) [inline src/tests/App.test.js]: I think this might need to change to `'../pages/App'` given the new file structure you introduced

**Pull request merged**